addons/knowledge/models/res_users.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, api
+
+
+class Users(models.Model):
+    _name = 'res.users'
+    _inherit = 'res.users'
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        users = super(Users, self).create(vals_list)
+        users.filtered(lambda user: not user.share)._generate_tutorial_articles()
+        return users
+
+    def _generate_tutorial_articles(self):
+        articles_to_create = []
+        article_template = self.env.ref('knowledge.knowledge_article_welcome')
```

Comment:
Would say "raise if not found=False", and skip if removed. We don't have to crash if someone removed the data :) .